az-release-pipelines/entrypoint.sh

```diff
@@ -27,6 +30,41 @@ AZDEVOPS_URL="https://dev.azure.com/${AZURE_PIPELINE_ORGANIZATION}/"
 vsts configure --defaults instance=${AZDEVOPS_URL} project=${AZURE_PIPELINE_PROJECT}
     
 vsts login --token ${AZURE_PIPELINE_TOKEN}
+
+PIPELINES=$( vsts release definition list --name ${AZURE_PIPELINE_NAME} )
+PIPELINE_COUNT=$( echo ${PIPELINES} | jq length )
+
+if [ $PIPELINE_COUNT -eq 0 ]
+then
+   echo "No release definition found with name: ${AZURE_PIPELINE_NAME}". >&2
+   exit 1;
+fi
+
+COUNT=0
+echo ${PIPELINES} | jq .[]?.name -r | while read PIPELINE ; 
+do
+   if [ "$PIPELINE" =  "$AZURE_PIPELINE_NAME" ]; 
```

Comment:
Make sure that it is ignore case comparison

Reply:
I have verified. This is ignore case